test/powershell/Modules/Microsoft.PowerShell.LocalAccounts/Pester.Command.Cmdlets.LocalAccounts.LocalGroup.Tests.ps1

```diff
@@ -27,11 +27,7 @@ function VerifyFailingTest
     $script:ErrorActionPreference = "Stop"
 
     try {
-        & $sb
-        throw "Expected FullyQualifiedErrorId: $expectedFqeid"
-    }
-    catch {
-        $_.FullyQualifiedErrorId | Should Be $expectedFqeid
+        {& $sb} | Should -Throw -ErrorId $expectedFqeid
```

Comment:
I'm not sure if this is the best way to refactor this with the use of `try/finally` here to toggle `ErrorActionPreference`. There is a hand full of instances of this pattern in this file

Reply:
I believe we could do:
```c#
{
    $script:ErrorActionPreference = "Stop" 
    try {
        & $sb
    } finally {
         $script:ErrorActionPreference = $backupEAP 
    }
} Should -Throw -ErrorId $expectedFqeid 
```